fluster/decoders/gstreamer.py

```diff
@@ -900,3 +900,15 @@ def __init__(self) -> None:
         self.api = "SW"
         self.caps = self.caps + ",format=S16LE"
         super().__init__()
+        
+        
+@register_decoder
+class FluendoH264LCEVCDecoder(GStreamer10Video):
+    """LCEVC-H264 decoder for GStreamer 1.0"""
+
+    codec = Codec.H264
+    decoder_bin = "flulcevchwvah264dec"
+    provider = "Fluendo"
+    api = "HW"
+    hw_acceleration = False
+    name = f"{provider}-{codec.value}-{api}-LCEVC-Gst1.0"
```

Comment:
Seems we are not consistent here but i like the idea of having the name of plugin in front of Gst1.0 like in Fluendo-H.265-HW-hwvah265dec-Gst1.0, so lcevchwvah264dec in place of LCEVC here